cmd/ovirt-populator/ovirt-populator.go

```diff
@@ -166,12 +159,24 @@ func loadEngineConfig(secretName, engineURL, namespace string) engineConfig {
 	secret, err := clientset.CoreV1().Secrets(namespace).Get(context.TODO(), secretName, metav1.GetOptions{})
 	if err != nil {
 		klog.Fatal(err.Error())
+	}*/
+	user, err := os.ReadFile("/etc/secret-volume/user")
+	if err != nil {
+		klog.Error(err.Error())
```

Comment:
missing 'return' here and below?

Reply:
yeah, maybe we should change to Fatal as it was - then it should crash the pod. I changed it to Error for debugging purpose